content/arduino-cloud/10.education/02.templates/content.md

```diff
@@ -0,0 +1,143 @@
+
+- USB Cable - Ensure you have the right one for your board 
+
+- Software - Arduino Account 
+
+- Good Internet Connection 
+
+- Shared Space (optional - only when you are about to use the templates with your students)
+
+- Sensors and Actuators (optional)
+
+***Note - if you haven't created a shared space check out this [article](https://docs.arduino.cc/arduino-cloud/education/shared-spaces/). Ensure that you students have their individual accounts created and have been added to the shared space that you have created.***  
+
+For our tutorial, we will use the [Cloud Blink Template](https://app.arduino.cc/templates/cloud-blink) that will illustrate how to control an Arduino Board’s LED through a dashboard in the Arduino Cloud. This is a great template if you want to get started with the Arduino Cloud since it works with all boards. After you have completed the installation, you can modify the sketch and dashboard freely.
+
+## Choosing a template 
+
+Once you have logged into your account, Templates can be accessed under the **IoT templates** section from the sidebar. 
+
+![Sidebar IoT Templates](assets/sidebar.png)
+
+Here you will find templates across various categories and different boards. Click on the "Cloud Blink" Template. 
+
+![Cloud blink template](assets/select-cloud-blink.png)
+
+The template page will open, where you will find the description of what the project does together with essential information on hardware and electronics components.
+
+![Template description](assets/template-description.png)
+
+## Importing a Template 
+
+Templates can be imported easily by clicking on the import button from the template description page. The process is similar to that of uploading a sketch to the Arduino board.  Once the device is plugged in the USB port, click on **IMPORT TEMPLATE**  to import the chosen IoT template.  
+
+![Import template button](assets/import-button.png)
+
+Below the `Import Button`, you can find information on how many [Things](https://docs.arduino.cc/arduino-cloud/cloud-interface/things), [Variables](https://docs.arduino.cc/arduino-cloud/cloud-interface/variables) and [Dashboards](https://docs.arduino.cc/arduino-cloud/cloud-interface/dashboard-widgets) are associated with this template. The Arduino Cloud platform will handle the creation of these IoT components along with setting up the device and network for the selected template. 
+
+![Template creation](assets/template-creation.png)
+
+This process involves creating the necessary components and uploading the sketch (program) to your device:
+
+1. Configure your Arduino hardware and add it to the Cloud platform,
+2. Create a virtual representation of your IoT setup,
+3. Assign the right network credentials to your devices, 
+4. Build a dashboard with the right widgets.
+
+***Note - The process may take up to 5 minutes. Remind students not to unplug the device during this step or close their laptops during the import process.*** 
```

Comment:
```suggestion
***Note: The process may take up to 5 minutes. Remind students not to unplug the device during this step or close their laptops during the import process.*** 
```

Reply:
@g-castronovo why was this resolved and not fixed?

Reply:
Sorry I must have missed it. Done now